rally/README.md

```diff
@@ -0,0 +1,38 @@
+# Rally
+
+An app that manages personal finance data. It showcases custom tabs, custom painted widgets,
+and custom animations.
+
+**This sample is not currently in a finished state. We're in the process
+of building it out.**
+
+## Goals
+
+* Show how to customize a tab bar.
+* Show how to create reusable custom widgets with composition and custom painting.
+* Show how to create an app with tabs and child navigation screens.
+
+## The important bits
+
+### `/charts/*`
+
+These are the custom charts. The is a circle chart and line chart are custom painters,
+while the single vertical bar chart is a simple composition of boxes.
+
+### `/sections/*`
+
+These are the main sections for the tab views and the child screen with the details and line chart.
+The financial entity is a reusable screen that is the base for the accounts, bills, and budgets
+screens.
+
```

Comment:
Add a notice section like:

```
#### Notice

* Theming is passed as a parameter in the constructor of `MaterialApp` (`theme:`).
* `debugShowCheckedModeBanner` is set to `false` so you can examine the entire screen as if it were a production app.
* `SafeArea` is the first widget in the `Scaffold`'s body. This corrects layout on screens with notches and virtual
    home buttons (like iPhone X+).
* The entire newspaper article is wrapped in a `SingleChildScrollView` widget which allows it to scroll instead of
    overflowing.
* The `Text` widget with text ' ¬ ' has a `TextStyle` that changes one parameter of an inherited `TextStyle` using
    `.apply().
* The `Text` widget with text 'Connor Eghan' has a `TextStyle` created explicitly instead of inheriting from theming.
* You can break up long strings in your source files by putting them on multiple lines.
* Fonts are imported with multiple files expressing their weights (Bold, Light, Medium, Regular) but are accessed by a
    `FontWeight` widget by number like 'FontWeight.w800' for Merriweather-Bold.ttf.
```

Reply:
Done, but will add more as more of the app gets checked in.